chart-modules/common/locale/all.json

```diff
@@ -564,5 +564,23 @@
       "ja-JP": "99 番目の百分位数",
       "ru-RU": "99-й процентиль"
     }
+  },
+  "properties.axis.startAt": {
```

Comment:
there should be no need to add properties strings here as they are only used inside sense-client

Reply:
Ok, I will remove it